src/styles/header.scss

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: block;
+  margin: 0 10px;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.nav__link {
+  display: block;
+  font-size: 12px;
+  text-decoration: none;
+  line-height: 60px;
+  color: #060b35;
+  transition: all 0.3s;
```

Comment:
Ensure that the `transition` property specifies which properties you want to transition, rather than using `all`, to avoid unintended side effects and improve performance. For example, `transition: color 0.3s;` if you only want to transition the color.